src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -966,8 +966,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                    Me.OptionExplicit = other.OptionExplicit AndAlso
                    Me.OptionCompareText = other.OptionCompareText AndAlso
                    Me.EmbedVbCoreRuntime = other.EmbedVbCoreRuntime AndAlso
-                   Me.SuppressEmbeddedDeclarations = other.SuppressEmbeddedDeclarations AndAlso
-                   If(Me.ParseOptions Is Nothing, other.ParseOptions Is Nothing, Me.ParseOptions.Equals(other.ParseOptions))
+                   Me.SuppressEmbeddedDeclarations = other.SuppressEmbeddedDeclarations AndAlso Me.ParseOptions.Equals(other.ParseOptions)
```

Comment:
>Me.SuppressEmbeddedDeclarations = other.SuppressEmbeddedDeclarations AndAlso Me.ParseOptions.Equals(other.ParseOptions) [](start = 19, length = 119)

Please follow the original style keeping condition for each property on a separate line.